test/boolean_test.cpp

```diff
@@ -371,1007 +371,3 @@ TEST(Boolean, Precision2) {
-       -1, 0, 0, 3,   //
-       0, -1, 0, 7},  //
-      {1, 0, 0, 3,    //
-       0, 0, 1, 4,    //
-       0, -1, 0, 6},  //
-      {1, 0, 0, 4,    //
-       0, 0, 1, 4,    //
-       0, -1, 0, 6},  //
-  };
-  float arrays2[][12] = {
-      {1, 0, 0, 3,    //
-       0, 0, 1, 2,    //
-       0, -1, 0, 6},  //
-      {1, 0, 0, 4,    //
-       0, 1, 0, 3,    //
-       0, 0, 1, 6},   //
-
-      {0, 0, 1, 2,    //
-       -1, 0, 0, 2,   //
-       0, -1, 0, 7},  //
-      {1, 0, 0, 3,    //
-       0, 1, 0, 3,    //
-       0, 0, 1, 7},   //
-      {1, 0, 0, 2,    //
-       0, 1, 0, 3,    //
-       0, 0, 1, 7},   //
-      {1, 0, 0, 1,    //
-       0, 1, 0, 3,    //
-       0, 0, 1, 7},   //
-      {1, 0, 0, 3,    //
-       0, 1, 0, 4,    //
-       0, 0, 1, 7},   //
-      {1, 0, 0, 3,    //
-       0, 1, 0, 5,    //
-       0, 0, 1, 6},   //
-      {0, 0, 1, 3,    //
-       -1, 0, 0, 4,   //
-       0, -1, 0, 6},  //
-  };
-
-  Manifold m1;
-  for (auto& array : arrays1) {
-    glm::mat4x3 mat;
-    for (const int i : {0, 1, 2, 3}) {
-      for (const int j : {0, 1, 2}) {
-        mat[i][j] = array[j * 4 + i];
-      }
-    }
-    m1 += rod.Transform(mat);
-  }
-
-  Manifold m2;
-  for (auto& array : arrays2) {
-    glm::mat4x3 mat;
-    for (const int i : {0, 1, 2, 3}) {
-      for (const int j : {0, 1, 2}) {
-        mat[i][j] = array[j * 4 + i];
-      }
-    }
-    m2 += rod.Transform(mat);
-  }
-  m1 += m2;
-
-  EXPECT_TRUE(m1.MatchesTriNormals());
-  EXPECT_LE(m1.NumDegenerateTris(), 12);
-}
-
-TEST(Boolean, Cubes) {
-  Manifold result = Manifold::Cube({1.2, 1, 1}, true).Translate({0, -0.5, 0.5});
-  result += Manifold::Cube({1, 0.8, 0.5}).Translate({-0.5, 0, 0.5});
-  result += Manifold::Cube({1.2, 0.1, 0.5}).Translate({-0.6, -0.1, 0});
-
-  EXPECT_TRUE(result.MatchesTriNormals());
-  EXPECT_LE(result.NumDegenerateTris(), 0);
-  auto prop = result.GetProperties();
-  EXPECT_NEAR(prop.volume, 1.6, 0.001);
-  EXPECT_NEAR(prop.surfaceArea, 9.2, 0.01);
-
-#ifdef MANIFOLD_EXPORT
-  if (options.exportModels) ExportMesh("cubes.glb", result.GetMesh(), {});
-#endif
-}
-
-TEST(Boolean, NoRetainedVerts) {
-  Manifold cube = Manifold::Cube(glm::vec3(1), true);
-  Manifold oct = Manifold::Sphere(1, 4);
-  EXPECT_NEAR(cube.GetProperties().volume, 1, 0.001);
-  EXPECT_NEAR(oct.GetProperties().volume, 1.333, 0.001);
-  EXPECT_NEAR((cube ^ oct).GetProperties().volume, 0.833, 0.001);
-}
-
-TEST(Boolean, PropertiesNoIntersection) {
-  MeshGL cubeUV = CubeUV();
-  Manifold m0(cubeUV);
-  Manifold m1 = m0.Translate(glm::vec3(1.5));
-  Manifold result = m0 + m1;
-  EXPECT_EQ(result.NumProp(), 2);
-  RelatedGL(result, {cubeUV});
-}
-
-TEST(Boolean, MixedProperties) {
-  MeshGL cubeUV = CubeUV();
-  Manifold m0(cubeUV);
-  Manifold m1 = Manifold::Cube();
-  Manifold result = m0 + m1.Translate(glm::vec3(0.5));
-  EXPECT_EQ(result.NumProp(), 2);
-  RelatedGL(result, {cubeUV, m1.GetMeshGL()});
-}
-
-TEST(Boolean, MixedNumProp) {
```

Comment:
From `TEST(Boolean, Cubes)` to here look like fairly simple tests - perhaps move them back here from the complex test file?

Reply:
Done

Reply:
sorry, I misunderstood. fixed it and move some other simple tests also.

Reply:
Perfect, and thanks for finding a few more I missed!